providers/instantly/delete.go

```diff
@@ -0,0 +1,39 @@
+package instantly
+
+import (
+	"context"
+
+	"github.com/amp-labs/connectors/common"
+)
+
+// Delete removes object. As of now only removal of Campaigns is allowed.
+func (c *Connector) Delete(ctx context.Context, config common.DeleteParams) (*common.DeleteResult, error) {
+	if len(config.ObjectName) == 0 {
+		return nil, common.ErrMissingObjects
+	}
+
+	if len(config.RecordId) == 0 {
+		return nil, common.ErrMissingRecordID
+	}
+
+	if !supportedObjectsByDelete.Has(config.ObjectName) {
+		// Removing campaign is the only to be supported at this time.
+		// https://api.smartlead.ai/reference/delete-campaign
+		return nil, common.ErrOperationNotSupportedForObject
+	}
+
+	url, err := c.getURL("custom-tag", config.RecordId)
```

Comment:
we shouldn't hard code this. I don't think you can delete campaign with custom-tag either. Campaign has id.

Reply:
I messed up the method documentation. I didn't update it.
The delete only works on `tags` which they call `custom-tag`.

It has nothing to do with campaigns. Speaking of campaigns they cannot be removed or created programatically.